src/Core/tests/DeviceTests/Stubs/LayoutManagerStub.cs

```diff
@@ -12,7 +12,7 @@ public Size ArrangeChildren(Rect bounds)
 
 		public Size Measure(double widthConstraint, double heightConstraint)
 		{
-			return new Size(widthConstraint, heightConstraint);
+			return new Size(widthConstraint - 1, heightConstraint - 1);
```

Comment:
The -1's here are just to keep the stub layout manager from accidentally making the Layout's explicit height/width tests from passing.